src/components/Japan/JpWeather/JpHourlyWeather.jsx

```diff
@@ -0,0 +1,19 @@
+function JpHourlyWeather({ days }) {
+  if (days === '') {
+    return <div></div>;
```

Comment:
you can use return null
if you don't want render anything

https://react.dev/learn/conditional-rendering

Reply:
ㄱㅅ